### ISSUE TITLE
MaxCommitBatch/ MaxPreCommitBatch  is too high 

### ISSUE THREAD

**Issue by jennijuju** [filecoin-project/lotus#10612]
A couple protocol changes have resulted in PCB and PCA gas usage increase:
- FVM M1 introduce proper gas accounting and  has resulted in  the overall gas usage,
- [FIP 0045 introduced additional claim/allocation  operations ](https://github.com/filecoin-project/FIPs/discussions/535)for deal activation during PCB/PCA which increase the gas usage 

That being said, if user creates messages with  the previous MaxPreCommitBatch (256)  and MaxCommitBatch (819) - the message likely will fail due to `message execution failed: exit SysErrOutOfGas` as the block limit has reached.

its similar with #8986 #8982, however, again with higher impact from fip0045

we should 
- perform gas benchmarking and adjust the max values (this is set in protocol, we likely should adjust accordingly there as its protocol-impossible to include this many sectors in one message under the current block limit)
- can start with lower max fees in lotus for now

**Comment by arajasek** [comment]
Yes, I think it's time we do this. In addition to "making more sense", I think it's a good idea to have fewer messages taking up (potentially) an entire gas block limit. I think as protocol designers we want such messages to be rare (though they'll always be possible with the FEVM).

**Comment by arajasek** [comment]
Would suggest creating a separate actors issue for the first point.
